lib/prop/limiter.rb

```diff
@@ -54,18 +56,17 @@ def disabled(&block)
       # Returns true if the threshold for this handle has been reached, else returns false
       def throttle(handle, key = nil, options = {})
         options, cache_key = prepare(handle, key, options)
-        counter = reader.call(cache_key).to_i
+        counter = options[:strategy].counter(cache_key, options)
```

Comment:
`options.fetch(:strategy)` to make it blow up nicely when it is not there ... maybe `@strategy` ?